src/index.php

```diff
@@ -139,12 +139,6 @@
     ),
 ));
 
-$app['twig'] = $app->share($app->extend('twig', function($twig, $app) {
-    $engine = new \Aptoma\Twig\Extension\MarkdownEngine\MichelfMarkdownEngine();
-    $twig->addExtension(new \Aptoma\Twig\Extension\MarkdownExtension($engine));
-    return $twig;
-}));
-
```

Comment:
We don't need to load this extension into twig since we aren't using it anywhere.